bigbluebutton-html5/imports/ui/components/tooltip/component.jsx

```diff
@@ -23,7 +23,16 @@ class Tooltip extends Component {
     const expandedEl = tooltipTarget.parentElement.querySelector('[aria-expanded="true"]');
     const isTarget = expandedEl === tooltipTarget;
     if (expandedEl && !isTarget) return;
-    tip.set({ content: tooltipTarget.lastChild.innerText });
+
+    if (tooltipTarget.lastChild && tooltipTarget.lastChild.innerText) {
+      tip.set({ content: tooltipTarget.lastChild.innerText });
+    } else if (tooltipTarget.parentElement.nodeName.toLowerCase() === 'button') {
+      tip.set({ content: tooltipTarget.parentElement.lastChild.innerText });
+    } else if (tooltipTarget.parentElement.parentElement.nodeName.toLowerCase() === 'button') {
+      tip.set({ content: tooltipTarget.parentElement.parentElement.lastChild.innerText });
```

Comment:
using `.parentElement.parentElement` like this probably isn't the best idea.

What do you think about something like this instead?

    const findBtnLabel = node => {
       const { nodeName, lastChild, parentElement } = node;
       if (nodeName.toLowerCase() === "button") return lastChild.innerText;
       return findBtnLabel(parentElement);
    }

    tip.set({ content: findBtnLabel(tooltipTarget) });
    tip.show();

Reply:
I like the idea. Already pushed an update.